src/view/sidebar/sidebar.tsx

```diff
@@ -54,37 +49,51 @@ const SidebarComponent: FC<SidebarComponentProps> = ({
 				folderId={selectedFolder}
 				localStorageName="open_calendars_folders"
 				AccordionCustomComponent={FoldersComponent}
+				buttonFindShares={<SharesComponent key={'calendar-find-share'} />}
 				setSelectedFolder={setSelectedFolder}
 			/>
 			<Divider />
 			<Accordion items={[tagsAccordionItems]} />
-			<Divider />
-			<Accordion items={[sharesAccordionItems]} />
 		</Container>
 	);
 };
 const MemoSidebar: FC<SidebarComponentProps> = React.memo(SidebarComponent);
 
-const Sidebar: FC<SidebarProps> = ({ expanded }) => {
-	const folders = useRootsArray();
-	const dispatch = useAppDispatch();
-	const start = useRangeStart();
-	const end = useRangeEnd();
-	const query = useCheckedCalendarsQuery();
-
-	const folderItems = removeLinkFolders({ folders });
+const addFindSharesItem = (foldersAccordionItems: Array<Folder>): Array<Folder> =>
+	foldersAccordionItems?.length > 0
+		? [
+				{
+					...(foldersAccordionItems?.[0] ?? {}),
+					children: [
+						...(foldersAccordionItems?.[0]?.children ?? []),
+						{
+							id: 'find_shares',
+							// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+							// @ts-ignore
```

Comment:
Why the ts-ignore? If it is not avoidable a comment should be added

Reply:
To be completely honest it was already there, I tried to remove it but it was complaining about that key which should not exist in the Folder type (and it is right) but at the same time I guess it is used somewhere in the sidebarAccordionMui but I am not sure so I don't know what is the real reason for it to be there, that is why I didn't add anything there